VersionCheckTest.testVersionCheckParameters

```
java.lang.AssertionError: Values should be different. Actual: 0
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failEquals(Assert.java:185)
	at org.junit.Assert.assertNotEquals(Assert.java:161)
	at org.junit.Assert.assertNotEquals(Assert.java:175)
	at com.hazelcast.util.VersionCheckTest.testVersionCheckParameters(VersionCheckTest.java:58)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.maintenance-IbmJDK1.8/com.hazelcast$hazelcast/23/testReport/junit/com.hazelcast.util/VersionCheckTest/testVersionCheckParameters/

--- a/hazelcast/src/test/java/com/hazelcast/util/VersionCheckTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/util/VersionCheckTest.java
@@ -39,9 +39,9 @@ public class VersionCheckTest extends HazelcastTestSupport {
 
     @Test
     public void testVersionCheckParameters() throws Exception {
-        assertClusterSizeEventually(1, hz1);
         Node node1 = TestUtil.getNode(hz1);
         VersionCheck check = new VersionCheck();
+        sleepAtLeastMillis(1);
         Map<String, String> parameters = check.doCheck(node1, "test_version", false);
 
         assertEquals(parameters.get("version"), "test_version");

